mailet/api/pom.xml

```diff
@@ -55,11 +55,16 @@
             <version>1.7.5</version>
```

Comment:
Many other things can invalidate headers...

If we recompute it all the time, then we have no benefits...